src/Windowing/Silk.NET.Windowing.Common/Structs/VideoMode.cs

```diff
@@ -27,10 +28,51 @@ public VideoMode(int refreshRate)
         /// Refresh rate of the full screen window in Hz.
         /// </summary>
         public int? RefreshRate { get; }
+        
+        /// <summary>
+        /// Aspect ratio of the full screen window.
+        /// </summary>
+        public Vector2D<int>? AspectRatio { get; }
 
         /// <summary>
         /// The default video mode. This uses the window size for resolution and doesn't care about other values.
         /// </summary>
         public static VideoMode Default => new VideoMode();
+
+        private static int GreatestCommonFactor(int value1, int value2)
```

Comment:
Interesting article: https://lemire.me/blog/2013/12/26/fastest-way-to-compute-the-greatest-common-divisor/

Reply:
Interesting, definitely something I will look to implement soon.